functional_testing/Openvcloud/RESTful/testcases/cloudbroker/test01_accounts.py

```diff
@@ -0,0 +1,306 @@
+from testcases import *
+from nose_parameterized import parameterized
+import random, unittest
+#from framework.api.client import Client
+
+
+
+class Test(TestcasesBase):
+
+    def setUp(self):
+        super().setUp()
+        self.user="{}@itsyouonline".format(self.whoami) 
```

Comment:
username should be passed in config file in this format ```username@itsyouonine```
so remove @itsyouonline and just use self.whoami